packages/block/CHANGELOG.md

```diff
@@ -6,6 +6,194 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+
+Both builds have respective separate entrypoints in the distributed `package.json` file.
+
+A CommonJS import of our libraries can then be done like this:
+
+```typescript
+const { Chain, Common } = require('@ethereumjs/common')
+const common = new Common({ chain: Chain.Mainnet })
+```
+
+And this is how an ESM import looks like:
+
+```typescript
+import { Chain, Common } from '@ethereumjs/common'
+const common = new Common({ chain: Chain.Mainnet })
+```
+
+Using ESM will give you additional advantages over CJS beyond browser usage like static code analysis / Tree Shaking which CJS can not provide.
+
+Side note: along this transition we also rewrote our whole test suite (yes!!!) to now work with [Vitest](https://vitest.dev/) instead of `Tape`.
+
+### Buffer -> Uint8Array
+
+With this releases we remove all Node.js specific `Buffer` usages from our libraries and replace these with [Uint8Array](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Uint8Array) representations, which are available both in Node.js and the browser (`Buffer` is a subclass of `Uint8Array`). While this is a big step towards interoperability and browser compatibility of our libraries, this is also one of the most invasive operations we have ever done, see the huge changeset from PR [#2566](https://github.com/ethereumjs/ethereumjs-monorepo/pull/2566) and [#2607](https://github.com/ethereumjs/ethereumjs-monorepo/pull/2607). 😋
+
+We nevertheless think this is very much worth it and we tried to make transition work as easy as possible.
+
+#### How to upgrade?
+
+For this library you should check if you use one of the following constructors, methods, constants or types and do a search and update input and/or output values or general usages and add conversion methods if necessary:
+
+```typescript
+// header
+BlockHeader.fromHeaderData(headerData: HeaderData = {}, opts: BlockOptions = {})
+BlockHeader.fromRLPSerializedHeader(serializedHeaderData: Uint8Array, opts: BlockOptions = {})
+BlockHeader.fromValuesArray(values: BlockHeaderBytes, opts: BlockOptions = {})
+new BlockHeader() // deprecated
+BlockHeader.raw(): BlockHeaderBytes
+BlockHeader.hash(): Uint8Array
+BlockHeader.cliqueExtraVanity(): Uint8Array // clique PoA only
+BlockHeader.cliqueExtraSeal(): Uint8Array // clique PoA only
+BlockHeader.serialize(): Uint8Array
+
+// block
+Block.fromBlockData(blockData: BlockData = {}, opts?: BlockOptions)
+Block.fromRLPSerializedBlock(serialized: Uint8Array, opts?: BlockOptions)
+Block.fromValuesArray(values: BlockBytes, opts?: BlockOptions)
+new Block() // deprecated
+Block.raw(): BlockBytes
+Block.hash(): Uint8Array
+Block.serialize(): Uint8Array
+```
+
+We have added helper methods for "Buffer -> Uint8Array" conversions in the [@ethereumjs/util](https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/util) `bytes` module, see the respective README section for guidance.
```

Comment:
I don't think this is 100% accurate. There are no buffer to bytes conversion methods in util and longer, or did I miss them somewhere?

Reply:
Yeah seconding @acolytec3 's point, to my knowledge we have no helper method that accepts Buffer as input. Would perhaps be more accurate to write that:
" We have converted existing Buffer conversion methods to Uint8Array conversion methods [...]  " ?

Reply:
Ah, yes, I already stumbled upon this upon writing but then I though "ah, hopefully they'll get it" 😂.

Will take the version from Gabriel, yes, that's more accurate.